### ISSUE TITLE
ubuntu lts 22.04 - php8.1 - 2.1.0-beta6 - doesn't work in subfolder

### ISSUE THREAD

**Issue by kiwi-x** [TasmoAdmin/TasmoAdmin#616]
TLDR;

tasmoadmin 2.1.0-beta6 doesn't work if it is installed in a subfolder of the webserver.

Long , maybe boring story :)

I updated my ubutnu to latest lts (from 20.04/php7.4 to 22.04/php8.1).
Now tasmoadmin does't run anymore. So I updated to tasmoadmin 2.1.0-beta6.
After npm install and composer install tasmoadmin looks like it still doesn't work.
I see a lost of 404 errors 'cause all script are referenced absolutly to the root folder of the webserver instead to the installation folder (relative?) to the installation folder.

I've installed tasmoadmin on my webserver in a subfolder like https://mydomain/tasmoadmin/ .
Latest 2.0 worked fine in subfolder.

Now I setup another webserver in a container with alpine linux and php7.4 and moved my tasmoadmin 1.8 (not 2.0 like I wrote here first) installation there.

I just want to mention this and maybe you can modify tasmoadmin to work in a subfolder, too.

Cheers
kiwi-x

**Comment by inverse** [comment]
- What web server are you using? 

- What does the web server config look like for Tasmoadmin?

- What version of Tasmoadmin was working?

**Comment by kiwi-x** [comment]
Hi,

I used: Apache 2 with php 7.4 on Ubuntu 20.04 LTS and/or Alpine Linux
My config isn't special, - allow override all for DocumentRoot/tasmoadmin
Tasmoadmin v1.8.0 (I coorected version in initial post)

Greatings
kiwi-x

**Comment by inverse** [comment]
Think I figured it out - I wasnt passing the base URL into the asset resolution logic for things mounted from node_modules.

Should be fixed in https://github.com/TasmoAdmin/TasmoAdmin/releases/tag/v2.1.0-beta7

Let me know

**Comment by kiwi-x** [comment]
Good morning,

I just installed the new beta and it looks like anythign is working on ubuntu 22.04, apache2, php8.1!

Great job and many thanks to fix it.

Cheers
Kiwi-X

**Comment by inverse** [comment]
Glad to hear!
